spec/controllers/creators_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe CreatorsController, type: :controller do
+
+  describe "GET#search" do
+    let (:vimeo_search_params) {
```

Comment:
Because this is not a single line, I'd recommend using `do` and `end` instead of `{` and `}` for the `let` block. It's much easier to see where it starts/ends with that syntax.